### PR TITLE
Issue #4410: whitespace after @

### PR DESCRIPTION
**Pull request by sergileon**
Issue https://github.com/checkstyle/checkstyle/issues/4410. I continue [PR](https://github.com/checkstyle/checkstyle/issues/4412)

### CODE REVIEW

File: config/checkstyle_checks.xml [hunk 1]
```diff
@@ -485,6 +485,7 @@
     <module name="NoLineWrap"/>
     <module name="NoWhitespaceAfter">
       <property name="tokens" value="ARRAY_INIT"/>
+      <property name="tokens" value="AT"/>
```
Comment (romani) [inline config/checkstyle_checks.xml]: please add this token to google_style too.
It is not clearly defined in style guide but, it is nonsense, most engineers even do not know that space is possible by syntax.

Comment (rnveach) [reply config/checkstyle_checks.xml]: As you add this to google config, please also add it to google's tests under `/src/it`.

Comment (sergileon) [reply config/checkstyle_checks.xml]: @romani and @rnveach in `google_check.xml `is no description module `NowhiteSpaceAfter`. In `/src/it `is no the tests for `NoWhitespaceAfter`. Are you sure what it need for current issue? I think what is task for new issue.

Comment (romani) [reply config/checkstyle_checks.xml]: No need to add it, fact that it is missed is unrelated issue.

**Comment by romani** [comment]
@sergileon , please share diff report

**Comment by sergileon** [comment]
@romani , [diff report](https://sergileon.github.io/4410/index.html)

**Comment by romani** [comment]
@rnveach , please finalize

File: config/checkstyle_checks.xml (hunk 1 above)
Comment (rnveach) [reply config/checkstyle_checks.xml]: @romani Does this warrant a new issue to add it?

**Comment by rnveach** [comment]
@romani We are not running regression for this?

**Comment by rnveach** [comment]
Merged.

**Pull request merged**